moveit_ros/planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -36,9 +36,11 @@
 
 #include <moveit/robot_model_loader/robot_model_loader.h>
 #include <moveit/profiler/profiler.h>
-#include <ros/ros.h>
+#include "rclcpp/rclcpp.hpp"
 #include <typeinfo>
 
+rclcpp::Logger LOGGER = rclcpp::get_logger("planning").get_child("robot_model_loader");
```

Comment:
We should be able to add `const` after https://github.com/ros2/rclcpp/pull/680. Not proposing any change for now to avoid over-complicating CI